src/main/java/frc/robot/RobotContainer.java

```diff
@@ -245,20 +243,20 @@ public void initAuton(Constants.Auton mode) {
     boolean success = true;
     switch (mode) {
       case TEST:
-        if (driveBaseSubsystem != null) {
+        /* if (driveBaseSubsystem != null) {
           autonModes = new AutonModes(driveBaseSubsystem);
         } else {
           success = false;
         }
-        break;
+        break;*/
       case PUSHTAXI: // the two taxis
       case INTAKETAXI:
-        if (driveBaseSubsystem != null && intakeSubsystem != null && cdsSubsystem != null) {
+        /* if (driveBaseSubsystem != null && intakeSubsystem != null && cdsSubsystem != null) {
           autonModes = new AutonModes(driveBaseSubsystem, intakeSubsystem, cdsSubsystem);
         } else {
           success = false;
         }
-        break;
+        break;*/
```

Comment:
Lets fix this instead of leaving commented

Reply:
Do you want me to remove the second constructor? Or is there a different fix we should use

Reply:
Actually just delete these lines and always use the constructor with all subsystems

Reply:
Sounds good, deleting it all